vignettes/sidebars.Rmd

```diff
@@ -58,215 +87,398 @@ iframe, .bslib-page-fill {
   border: var(--bs-border-width) solid var(--bs-border-color);
   margin-bottom: 4rem;
   box-shadow:
-    rgba(0, 0, 0, 0.25) 0px 54px 55px, 
-    rgba(0, 0, 0, 0.12) 0px -12px 30px, 
-    rgba(0, 0, 0, 0.12) 0px 4px 6px, 
-    rgba(0, 0, 0, 0.17) 0px 12px 13px, 
+    rgba(0, 0, 0, 0.25) 0px 54px 55px,
+    rgba(0, 0, 0, 0.12) 0px -12px 30px,
+    rgba(0, 0, 0, 0.12) 0px 4px 6px,
+    rgba(0, 0, 0, 0.17) 0px 12px 13px,
     rgba(0, 0, 0, 0.09) 0px -3px 5px;
 }
+
+iframe {
+  margin-bottom: calc(55px + 2rem);
+}
+
+iframe.resizable {
+  resize: vertical;
+  min-height: 300px;
+}
+
+.card-header.bg-info code {
+  background: none;
+  color: hsl(209deg, 85%, 25%);
+}
 ```
 
-### Sidebar layouts
+::: lead
+Sidebar layouts in web interfaces allow your users to easily access filters, settings and other inputs alongside the interactive features they control.
+In this article, you'll learn how to create sidebar layouts with bslib.
+:::
+
+## Introduction
+
+To create a sidebar layout, provide a `sidebar()` to the `sidebar` argument of one of the following functions.
```

Comment:
You're probably aware of this, and I'm not sure how best to address it, but I find this sentence a little bit awkward (since you have to keep reading through multiple paragraphs to see what functions you're referencing). Maybe you could mention/summarize them in a sentence or two?

Reply:
Hmm, maybe something like this?

```suggestion
To create a sidebar layout, provide a `sidebar()` to the `sidebar` argument to `layout_sidebar()` or any of the `navs_` or `page_` functions that support sidebars.
```